Appraisals

```diff
@@ -3,3 +3,11 @@
 appraise "administrate-0.19" do
   gem "administrate", "0.19.0"
 end
+
+appraise "administrate-0.20" do
+  gem "administrate", "0.20.0"
+end
+
+appraise "administrate-main" do
+  gem "administrate", git: "https://github.com/thoughtbot/administrate", branch: "main"
+end
```

Comment:
I think these changes are helpful for this PR, as they couldn't be there previously … but I intend to pull this out before we merge it.